lib/msal-node/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# User-specific files
```

Comment:
This file addition appears in both this PR and #1325, potential MCs

Reply:
Thanks @jmckennon. This branch will be the base for #1325 so we are covered.